### DISCUSSION TITLE
Helsinki Model 54189 (2021 Brown Egg)

### DISCUSSION THREAD

**Discussion by dan88uk** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#297] [Q&A]
Hey everyone.
I seem to be having the same issue as others on here. 
I have a brown egg, manufactured 15/03/2021, model 54189, 6 wire, 240V version.
I built the v1 pcb, included the 10k ohm resistor and also added the 560ohm resistors on pins 3, 4 and 5 for the output.
When I install the pcb into the egg, I switch on the socket, can hear a solenoid kick in and a very very quiet beep and that's it. No display, no buttons work, and the ESP led doesn't light up.
Can anyone suggest anything else? I've tried the both models in the model.h file.
I look forward to any suggestions, really excited to get this working. 
@visualapproach awesome project!

**Comment by visualapproach** [comment]
Hi! Actually there is 3 or 4 versions to choose from in model.h depending on when you downloaded the files. Maybe not important. If there is an option to choose PCB make sure you select the v1. I don't think you need the R560s in that model. Have you tried without?

**Reply by dan88uk** [reply]
Hey! I'll have a look at choosing a different model and try again. Originally tried without the 560 resistors and got the same result unfortunately. I'll let you know how I get on when I change the model in model.h. For reference, this is my 2021 Helsinki egg (warranty replacement) ![image](https://user-images.githubusercontent.com/79228597/173228144-15388b3f-aedc-45df-b78e-3adc24d928cd.jpeg)

**Reply by visualapproach** [reply]
That's same model as I have and mine is PRE2021. No resistors. Try PRE2021 if you haven't

**Reply by dan88uk** [reply]
I'll give that a go...

To be fair, I did wonder if I'd burnt out the level converter... my first time soldering since GCSE tech 15 odd years ago!

**Reply by dan88uk** [reply]
So, a bit of an update... I made another pcb, this time leaving out the 560ohm resistors and I got the same result. I used the pre2021 model as well. 
Can anyone suggest anything else to try?

**Reply by dan88uk** [reply]
Forgot to say after the second attempt, I also removed the optional 10kohm resistor to see if that made any difference and same result... ;(

**Reply by dan88uk** [reply]
![image](https://user-images.githubusercontent.com/79228597/176540234-b2ce1565-c2e9-47a6-a307-bac49f8336dd.png)

![image](https://user-images.githubusercontent.com/79228597/176540268-2b760722-c72e-4e88-9961-d77fd103ebc3.png)

These are the components I bought... they're the correct ones aren't they?

**Reply by visualapproach** [reply]
Yes they should work. Maybe you have poor cables or soldering? If you feel adventurous read this #312 (one less thing to go wrong)

**Reply by dan88uk** [reply]
Maybe I'll look at getting some replacement parts from a different supplier. Thanks for your help :)

**Reply by dan88uk** [reply]
@visualapproach I've just been thinking about my pcb issue whilst at work and realised... could I have my 6 wire male / female cable soldered in the incorrect orientation!?!

**Reply by dan88uk** [reply]
@visualapproach 
So..... I've just potentially found the problem...! Spot the mistake.... ![image](https://user-images.githubusercontent.com/79228597/176763430-6e33d506-8eda-4db7-9b06-1ce2baa98ae8.jpeg)

**Reply by visualapproach** [reply]
Yup. Gender matters in this case.
